osgtest/tests/test_60_singularity.py

```diff
@@ -0,0 +1,53 @@
+import os
+import osgtest.library.core as core
+import osgtest.library.files as files
+import osgtest.library.osgunittest as osgunittest
+import tempfile
+
+class TestSingularity(osgunittest.OSGTestCase):
+
+    __check_path = '/cvmfs/cms.cern.ch/cmsset_default.sh'
+    __cvmfs_image = '/cvmfs/singularity.opensciencegrid.org/opensciencegrid/osg-wn:3.3-el6'
+
+    def mountSingularityCVMFSRepo(self, repo):
+        command = ('mkdir', '-p', /cvmfs + repo)
+        status, stdout, stderr = core.system(command, False)
+        if status != 0:
+            self.fail("failed to mkdir /cvmfs/%s" % repo)
+
+        command = ('mount', '-t', 'cvmfs', 'repo', '/cvmfs/' + repo)
+        status, stdout, stderr = core.system(command, False)
+        if status != 0:
+            self.fail("failed to mount: %s, error: %s" % (repo,stdout))
+                               
+
+    def test_01_singularity(self):
+        core.skip_ok_unless_installed('singularity-runtime')
+        core.skip_ok_unless_installed('cvmfs')
+        singularity_repo = 'singularity.opensciencegrid.org'
+        if core.el_release() <= 6:
+             self.mountSingularityCVMFSRepo(singularity_repo)
+        core.state['cvmfs.mounted'] = False
+
+        command = ('ls', '/cvmfs')
+        status, stdout, stderr = core.system(command, False)
+        file_exists = os.path.exists('/cvmfs')
+        self.assert_(file_exists, 'Cvmfs mount point missing')
+        core.state['cvmfs.mounted'] = True
```

Comment:
Can you instead just check the value of `core.state['cvmfs.mounted']` since test_38_cvmfs:test_02_cvmfs already performs this check?